packages/nodes-base/tsconfig.benchmark.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Could we do a on-the-fly compilation of TS code (like ts-jest does), instead of needing yet another tsconfig?